lib/event-subscribers.loader.ts

```diff
@@ -128,9 +130,10 @@ export class EventSubscribersLoader
     listenerMethod(
       event,
       async (...args: unknown[]) => {
-        const contextId = ContextIdFactory.create();
+        const request = this.getRequestFromEventPayload(args);
+        const contextId = ContextIdFactory.getByRequest({ payload: request });
```

Comment:
This change is a conscious choice I have made.  
There is no access to the original request object at this time, so in order to decide the request object for the strategy, I decided to use the request payload.  
I prefixed with the payload key, since the request object is of expected type `Record<string, any>`, while payload can also be any non-object or array.